docs/ops/sort/MulticlassNMS_8.md

```diff
@@ -0,0 +1,176 @@
+##  MulticlassNms<a name="MulticlassNms"></a> {#openvino_docs_ops_sort_MulticlassNms_8}
+
+**Versioned name**: *MulticlassNms-8*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: *MulticlassNms* performs multi-class non maximum suppression of the boxes with predicted scores.
+
+**Detailed description**: *MulticlassNms* is a multi-phase operation. It performs non maximum suppression algorithm as described below:
+
+1.  Let `B = [b_0,...,b_n]` be the list of initial detection boxes, `S = [s_0,...,s_N]` be  the list of corresponding scores.
+2.  Let `D = []` be an initial collection of resulting boxes. Let `adaptive_threshold = iou_threshold`.
+3.  If `B` is empty then go to step 9.
+4.  Take the box with highest score. Suppose that it is the box `b` with the score `s`.
+5.  Delete `b` from `B`.
+6.  If the score `s` is greater or equal than `score_threshold`  then add `b` to `D` else go to step 9.
+7.  If `nms_eta < 1` and `adaptive_threshold > 0.5`, update `adaptive_threshold *= nms_eta`.
+8.  For each input box `b_i` from `B` and the corresponding score `s_i`, set `s_i = 0` when `iou(b, b_i) > adaptive_threshold`, and go to step 3.
+9.  Return `D`, a collection of the corresponding scores `S`, and the number of elements in `D`.
+
+This algorithm is applied independently to each class of each batch element. The total number of output boxes for each class must not exceed `nms_top_k`. Boxes of `background_class` are skipped and thus eliminated. After NMS step, it selects at most `keep_top_k` scoring candidate boxes per batch element.
+
+**Attributes**:
+
+* *sort_result_byclass*
```

Comment:
add `_` character between "by" and "class".

Or may be we should create a more generic attribute like in docs/ops/sort/TopK_3.md specification to specify the sort field?